### PR TITLE
fix userCamera flag error with old version files

### PR DESCRIPTION
**Pull request by liamcindy**
@pandamicro  Please review this PR, thanks.

### CODE REVIEW

File: cocos/editor-support/cocostudio/WidgetReader/UserCameraReader/UserCameraReader.cpp [hunk 1]
```diff
@@ -133,7 +133,9 @@ namespace cocostudio
             }
             else if (attriname == "CameraFlagData")
             {
-                cameraFlag = atoi(value.c_str());
+                int flag = atoi(value.c_str());
+                if (flag != 0)
+                    cameraFlag = atoi(value.c_str());
```
Comment (WenhaiLin) [inline cocos/editor-support/cocostudio/WidgetReader/UserCameraReader/UserCameraReader.cpp:138]: Bad! You should not convert the string twice.

``` cpp
cameraFlag = flag;
```

Comment (pandamicro) [inline cocos/editor-support/cocostudio/WidgetReader/UserCameraReader/UserCameraReader.cpp:138]: Oh, sorry, I haven't noticed, @liamcindy Please send another PR to fix it

Comment (liamcindy) [inline cocos/editor-support/cocostudio/WidgetReader/UserCameraReader/UserCameraReader.cpp:138]: Ok

**Comment by pandamicro** [comment]
Fixed at https://github.com/cocos2d/cocos2d-x/pull/13414
